source/isEmpty.js

```diff
@@ -4,7 +4,7 @@ import equals from './equals.js';
 
 
 /**
- * Returns `true` if the given value is its type's empty value; `false`
+ * Returns `false` if the given value is its type's empty value; `true`
```

Comment:
I think this was correct before? Return `true` if value matches type's empty value?

Reply:
oh, lol, i copy pasted and then updated the wrong one. Changing back